src/datepicker/datepicker-config.ts

```diff
@@ -10,12 +10,14 @@ import {NgbDateStruct} from './ngb-date-struct';
 @Injectable()
 export class NgbDatepickerConfig {
   dayTemplate: TemplateRef<DayTemplateContext>;
+  displayMonths = 1;
   firstDayOfWeek = 1;
   markDisabled: (date: NgbDateStruct, current: {year: number, month: number}) => boolean;
   minDate: NgbDateStruct;
   maxDate: NgbDateStruct;
   outsideDays: 'visible' | 'collapsed' | 'hidden' = 'visible';
   showNavigation = true;
+  showNavigationSelect = true;
```

Comment:
May I suggest:

`navigation: true / false (default: true)` completely show hide the navigation header
`quickNavigation: true / false (default: false)` activate the quick/rapid navigation with selects